kwanj-k/ride-my-way-API-V1
# #158576375 - Adds flask_bycrypt to hash passwords

### What does this PR do?
Stores password as  hash

### Description of Task to be completed
* Add flask bycrypt to store password as hash 
* Add a check password method for checking against password hash during login

### How should I test it?
 1. Clone the repo
  ```
  $ git clone https://github.com/kwanj-k/ride-my-way-API-V1.git
  $ cd into the cloned repo
  $ git checkout ft-password-hash-#158576375
  ```
Create a virtualenv and activate it::

    Virtualenv -p python3 .
    source bin/activate

Install the dependencies::

    pip install -r requirements.txt 


## Testing on Postman
Run the development server:
  ```
  $ flask run
  ```
Users data:
```
{
	"email":"athena.kay@hmail.com",
	"password":"zeusskay",
	"username":"kwanjkay"
}
```
Using rides data:

* Make a `POST` request to `/api/v1/auth/signup`
* Make a `POST` request to `/api/v1/auth/signin`

All the end-points should respond accordingly


### What PT stories?
[#158576375]https://www.pivotaltracker.com/story/show/158576375
